PayPal/src/main/java/com/braintreepayments/api/paypal/AmountBreakdown.kt

```diff
@@ -41,4 +42,27 @@ data class AmountBreakdown(
     val insuranceTotal: String?,
     val shippingDiscount: String?,
     val discountTotal: String?
-) : Parcelable
+) : Parcelable {
+
+    fun toJson(): JSONObject {
```

Comment:
Sorry I missed this in the initial review - can we mark this function as internal? The merchant shouldn't need access to this function.